sql/2020/03_Markup/pages_almanac_by_device_and_script_type_frequency.sql

```diff
@@ -0,0 +1,40 @@
+#standardSQL
+# pages almanac metrics grouped by device and element attribute use (frequency)
+
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
```

Comment:
Seems a bit overkill to have a whole function for one line of code?

Reply:
More about consistency. And a copy/paste. There was mention of making a function like this global so everyone uses the same pattern. I personally think we could drop the ROUND as it can cause accuracy issues and is not needed if people format in the sheets.